void test_issue_py_18();
